robot/server/server.go

```diff
@@ -41,19 +42,15 @@ type Server struct {
 	activeBackgroundWorkers sync.WaitGroup
 	cancelCtx               context.Context
 	cancel                  func()
-	moduleLogger            logging.Logger
 }
 
 // New constructs a gRPC service server for a Robot.
 func New(r robot.Robot) pb.RobotServiceServer {
 	cancelCtx, cancel := context.WithCancel(context.Background())
-	modLogger := r.Logger().Sublogger("module")
-	modLogger.SetLevel(logging.DEBUG)
```

Comment:
I started creating `Sublogger`s in the `ModuleLog` receiver on the server-side instead of a global one here. I also did _not_ set a DEBUG log level for them, was this line intentional or just for testing, @dgottlieb?

Reply:
This line was intentional. Otherwise a module set to debug level would send an module log GRPC request, but the `r.Logger` would filter it out.

Reply:
(I should have documented the intention there)

Reply:
Ah that makes sense, good catch! Added a similar line back in and explained its presence in a comment.